app/controllers/api/v1x2/portfolio_items_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1x2
+    class PortfolioItemsController < Api::V1x1::PortfolioItemsController
+      def show
+        portfolio_item = if params[:showDiscarded] == "true"
+                           model.with_discarded.discarded.find(params.require(:id))
+                         else
+                           model.find(params.require(:id))
+                         end
+
+        authorize(portfolio_item)
```

Comment:
The rest code is the same as v1x1. You should refactor the common code in v1x1 into a separate private method